subgraph/networks.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Can we use the regular network package here?

Reply:
yes, this is built from the networks package !